checker/src/org/checkerframework/checker/fenum/FenumVisitor.java

```diff
@@ -58,7 +59,7 @@ public Void visitSwitch(SwitchTree node, Void p) {
 
     @Override
     protected boolean checkConstructorInvocation(AnnotatedDeclaredType dt,
```

Comment:
Most of the other `checkXXX` methods have a void return type. Is anyone actually using the boolean returned by this function?